server/game/gameSystems/DrawSystem.ts

```diff
@@ -52,10 +52,17 @@ export class DrawSystem<TContext extends AbilityContext = AbilityContext> extend
             const contingentEvents = [];
             if (event.amount > event.player.drawDeck.length) {
                 const damageAmount = 3 * (event.amount - event.player.drawDeck.length);
+
+                // Here we generate a the damage event with a new context that contains just the player,
+                // this way the damage is attributed to the player and not the card that triggered the draw (or its controller).
+                // As per rules, the player that is drawing is also the player that is causing the damage and
+                // this is important for cards like Forced Surrender.
```

Comment:
Could you add a small note like `(FFG ruling confirms this)`. Unfortunately they haven't added the ruling into the wiki yet so we can't link it directly